cross/python3/patches/006-configure-fix-autodetection-for-unused-result.patch

```diff
@@ -0,0 +1,14 @@
+Fixes the mechanism used to detect if the no-unused-result flag is supported
+
+--- configure.ac.orig
++++ configure.ac
+@@ -1630,7 +1630,7 @@ yes)
+     *)
+     AC_MSG_CHECKING(if we can turn off $CC unused result warning)
+      ac_save_cc="$CC"
+-     CC="$CC -Wunused-result -Werror"
++     CC="$CC -Wno-unused-result -Werror"
```

Comment:
For some reason the compiler used for ppc853x platform did support the `-Wunused-result` flag but not the `-Wno-unused-result` one. The thing is it was trying, here, to identify whether the latter was supported.

Reply:
No, actually this was not a trial.
The point of this process was to detect automatically detect if the `-Wno-unused-result` switch was supported.
It turns out, that, for unknown reason, the ppc853x architecture compiled did support the `-Wunused-result` switch but not the `-Wno-unused-result` one.
So here I changed the logic to use the `-Wno-unused-result` switch to detect if it was supported.
Obviously I could change it only for the given architecture, but I thought that it was kinda weird.

Obviously, tell me if you'd rather go for the architecture switch. (I saw that, indeed, you could create a specific folder for architecture specific options).

Reply:
With a previous change I made to the toolchain you can then simply create a directory `patches/ppc853x` and drop the patch that is only specific to that `arch`.